### PR TITLE
Fixes 2 issues in master

### PR DESCRIPTION
**Pull request by jdalsem**
- markdown pages are not styled #8529 
- walled garden pages missing some styling in current master branch #8464

### CODE REVIEW

File: engine/lib/admin.php [hunk 1]
```diff
@@ -598,6 +598,8 @@ function _elgg_admin_plugin_screenshot_page_handler($pages) {
 function _elgg_admin_markdown_page_handler($pages) {
 	set_input('plugin_id', elgg_extract(0, $pages));
 	set_input('filename', elgg_extract(1, $pages));
+	
+	elgg_set_context('admin');
```
Comment (hypeJunction) [inline engine/lib/admin.php]: maybe push markdown context afterwards to be sure?

**Comment by hypeJunction** [comment]
LGTM

File: engine/lib/admin.php (hunk 1 above)
Comment (jdalsem) [inline engine/lib/admin.php]: do you mean to pop it? there is no markdown context...

Comment (hypeJunction) [inline engine/lib/admin.php]: No, push it. Setting admin, replaces the stack. Plugins don't know anymore
if markdown is part of the stack.

Comment (jdalsem) [inline engine/lib/admin.php]: markdown was never part of the stack...

File: views/default/page/walled_garden.php [hunk 2]
```diff
@@ -35,7 +35,7 @@
 	<div class="elgg-page-messages">
 		<?php echo $messages ?>
 	</div>
-	<div class="$wg_body_class">
+	<div class="<?php echo $wg_body_class; ?>">
```
Comment (ewinslow) [inline views/default/page/walled_garden.php]: You can use short tags.

**Comment by ewinslow** [comment]
LGTM

File: engine/lib/admin.php (hunk 1 above)
Comment (jdalsem) [inline engine/lib/admin.php]: not doing anything now about it, as this is the same behaviour as before the resources views implementation

**Pull request merged**